### DISCUSSION TITLE
Improving our soundness using the Fuzzer

### DISCUSSION THREAD

**Discussion by gideonred** [RecordReplay/devtools#6808] [Ideas]
During our Quality chat we discussed that our soundness (we don't contradict ourselves) is important. Jason mentioned adding more things to verify using the fuzzer. The question is, what are those things?

**Comment by bhackett1024** [comment]
I don't know if there's much we can do here automatically.  Anytime we see something in the devtools that looks wrong we should make a recording if possible, or failing that write down a session ID and a description of the problem.
